### PR TITLE
Extend explore-main-media switch

### PR DESCRIPTION
**Pull request by TBonnin**
## What does this change?
@SidRoberts, the owner of this switch, is off for 2 weeks. Extending it so he can make a decision when he comes back

## What is the value of this and can you measure success?
No failing build because of expired switch

## Tested in CODE?
No

### CODE REVIEW

**Comment by NataliaLKB** [comment]
I think you mean @SiAdcock.

Sorry @SidRoberts!

File: common/app/conf/switches/FeatureSwitches.scala [hunk 1]
```diff
@@ -21,7 +21,7 @@ trait FeatureSwitches {
     "If this switch is on, Explore template will be applied to explore articles. This template is part of a Membership Explore test",
     owners = Seq(Owner.withGithub("siadcock")),
```
Comment (NataliaLKB) [inline common/app/conf/switches/FeatureSwitches.scala]: Feel free to add me as an owner.

**Review by NataliaLKB — APPROVED** [review]
Thanks for doing this @TBonnin. 

I have asked for an update on this template from @blongden73. Last time I had an update it was unclear as to whether we were keeping or killing this. Hoping for an answer this time!

**Comment by TBonnin** [comment]
Thanks @NataliaLKB. I give you the ownership of this PR/branch 🎁 .
Feel free to close it if you get an answer from @blongden73 or merge it if not.

**Comment by NataliaLKB** [comment]
@blongden73 and I had a chat and this template is needed for a few weeks more for some UX testing. Afterwards a decision will be made as to whether to keep or kill.

**Pull request merged**